portfolio/src/main/webapp/script.js

```diff
@@ -69,6 +69,9 @@ function validatePassword(form) {
     }
 }
 
+function isLoggedIn() {
```

Comment:
I would either implement this or remove it.
(Or at the very least add a comment and a fake implementation like "return false")

Reply:
I am planning on implementing it in the next step.